lib/logger/src/helpers.ts

```diff
@@ -0,0 +1,65 @@
+import { ChildProcess } from 'child_process'
+import hookStd from 'hook-std'
+import { Transform } from 'stream'
+import type { Logger } from 'winston'
+import { ToolKitError } from '@dotcom-tool-kit/error'
+
+// winston will add a newline itself
+function trimTrailingNewline(message: string): string {
+  return message.endsWith('\n') ? message.slice(0, -1) : message
+}
+
+// This function hooks winston into console.log statements. Most useful for when
+// calling functions from external libraries that you expect will do their own
+// logging.
+export function hookConsole(logger: Logger, process: string): hookStd.Unhook {
+  const { unhook: unhookStd } = hookStd.stderr({ silent: true }, (output) => {
+    logger.info(trimTrailingNewline(output), { process })
+  })
+  const { unhook: unhookErr } = hookStd.stderr({ silent: true }, (output) => {
+    logger.warn(trimTrailingNewline(output), { process })
+  })
+
+  return () => {
+    unhookStd()
+    unhookErr()
+  }
+}
+
+// This function hooks winston into the stdout and stderr of child processes
+// that we have spawned forked. Useful for when you need to invoke a CLI tool.
+export function hookFork(logger: Logger, process: string, child: ChildProcess): void {
+  if (!child.stdout) {
```

Comment:
I didn't intend to omit `silent: true`: it makes the forked process pipe into our node process rather than inheriting the `stdout` and `stderr` stream from our parent process. Once you enable it Jest can see that it's no longer writing a terminal so automatically disables coloured ANSI output. Jest's [`--colors`](https://jestjs.io/docs/cli#--colors) doesn't seem to properly override the setting though. I'll keep investigating...

Reply:
the colors are back, but I see that there's [JestLocal][jest] in every new line when the test coverage is logged out, and also `jest process returned an error`, `null`at the end when any test fails

<img src="https://user-images.githubusercontent.com/20088576/151342369-249535eb-41f4-4e15-9171-fc9b8dcdbe42.png" width="500" height="250">

Reply:
somehow the comment I left got accidentally deleted, so adding it back - 

I just reran `npm run test` with your new changes, it's great that the color is back, though I see [JestLocal][jest] in every line and also `jest process returned an error` and `null` 
<img src="https://user-images.githubusercontent.com/20088576/151345488-8e5a3d5e-6773-4c61-bca0-9661e9e0ea43.png" width="400" height="200">